lib/money_market_account.rb

```diff
@@ -0,0 +1,48 @@
+module Bank 
+	class MoneyMarketAccount < Account 
+		attr_accessor :transaction_count
+
+		def initialize(account_id, balance, date_opened)
+			super
+			@min_balance = 10_000
+			@transaction_count = 0
+		end
+
+		def withdraw(withdraw_amount)	
+			if (@balance - withdraw_amount) <= min_balance && @transaction_count < 6
```

Comment:
Since you use this `6` in different places, I think it would make sense to use a constant for this so that if the transaction count ever changed, you would only need to change it in one place